index.html

```diff
@@ -3543,6 +3543,12 @@ <h3>Algorithm</h3>
           </ol>
         </li>
         <li>If <var>compact IRI</var> is not <code>null</code>, return <var>compact IRI</var>.</li>
+        <li class="changed">To ensure that the <a>absolute IRI</a> <var>var</var> is
+          not confused with a <a>compact IRI</a>,
+          if the <a data-cite="RFC3986#section-3.1">IRI scheme</a> of <var>var</var>
+          matches any term in <a>active context</a>,
```

Comment:
```suggestion
          matches any term in <a>active context</a>,
          and <var>var</var> has no <a datacite="https://tools.ietf.org/html/rfc3986#section-3.2">IRI authority</a> (beginning with double-forward-slash (<code>//</code>),
```

Reply:
Alternatively, we might remove the double-slash caveat, which was just one way of addressing this issue, and just use the scheme/term distinction. Otherwise, it tacitly allows the use of terms which are also IRI schemes.

Reply:
I strongly disagree. The double-slash "caveat" ensures that *only a subset* of absolute IRIs can potentially be confused with compact IRIs. I think we agreed, during the last meeting, that this ambiguity between absolute IRIs and compact IRIs, was unfortunate. Removing the double-slash rule would increase this ambiguity. And doing so, we increase the number of cases were compaction would fail with `iri confused with prefix`...

> Otherwise, it tacitly allows the use of terms which are also IRI schemes.

which might be absolutely valid ([example]). The problem is using them as prefixes...

[example]: https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22%40vocab%22%3A%22http%3A%2F%2Fexample.org%2FserviceOntology%23%22%2C%22protocol%22%3A%7B%22%40type%22%3A%22%40vocab%22%7D%2C%22http%22%3A%22http%3A%2F%2Fdbpedia.org%2Fresource%2FHypertext_Transfer_Protocol%22%2C%22https%22%3A%22http%3A%2F%2Fdbpedia.org%2Fresource%2FHTTPS%22%2C%22ftp%22%3A%22http%3A%2F%2Fdbpedia.org%2Fresource%2FFile_Transfer_Protocol%22%7D%2C%22%40type%22%3A%22Service%22%2C%22address%22%3A%22123.45.67.89%22%2C%22protocol%22%3A%22http%22%7D